### DISCUSSION TITLE
How to tweet with line breaks?

### DISCUSSION THREAD

**Discussion by dbeja** [plhery/node-twitter-api-v2#286] [Q&A]
Is there a way to post a tweet with line breaks?
I tried `\n` and `%0A` but I just get those chars on the tweet.

**Comment by IMXNOOBX** [comment]
Same here, I tried in a few ways and the only one that worked for me was writing the text normally. like in the image velow.
![image](https://user-images.githubusercontent.com/69653071/172155844-c9de89da-7a14-4760-9e8c-065f1d5318d4.png)

result: 
![image](https://user-images.githubusercontent.com/69653071/172156156-c2752c83-b4bc-4eb0-ac49-a7d11d72eeaa.png)

**Reply by dbeja** [reply]
Thanks! In the end, I've added the line breaks on PHP where I was getting the message text before processing here on the node with `chr(13) . chr(10)` but good to know how to do it in node.
